base/cvd/cuttlefish/host/commands/cvd/fetch/BUILD.bazel

```diff
@@ -5,6 +5,11 @@ package(
     default_visibility = ["//:android_cuttlefish"],
 )
 
+filegroup(
+    name = "host_package_symlinks_dev",
+    srcs = ["host_package_symlinks_dev.json"],
+)
```

Comment:
This file should be installed somewhere using `base/debian/cuttlefish-base.install`, so that it can be referenced from the packaged version.

Reply:
Removed this logic in favor of debian_substitution_marker